clients/go-ethereum_master/enode.sh

```diff
@@ -8,10 +8,10 @@
 
 # Immediately abort the script on any error encountered
 set -e
-echo "Trying to get enode."
 
-TARGET_RESPONSE=$(curl -s -X POST  -H "Content-Type: application/json"  --data '{"jsonrpc":"2.0","method":"admin_nodeInfo","params":[],"id":1}' "$HIVE_CLIENT_IP:8545" )
 
-echo "Got admin enode info response: $TARGET_RESPONSE"
+TARGET_RESPONSE=$(curl -s -X POST  -H "Content-Type: application/json"  --data '{"jsonrpc":"2.0","method":"admin_nodeInfo","params":[],"id":1}' "localhost:8545" )
```

Comment:
You could use the `ipc` file instead, then you wouldn't even need `curl`. `nc -u` (but you'd need `nmap-ncat` instead though )